### ISSUE TITLE
osmium-tools not building properly

### ISSUE THREAD

**Issue by tspoke** [door2door-io/osrm-express-server-demo#6]
Hello,

Thanks for the repository.
I tried to build the osmium tools docker image without success. 

First, I got this error :  

```
...
fetch http://dl-cdn.alpinelinux.org/alpine/v3.7/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.7/community/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/edge/testing/x86_64/APKINDEX.tar.gz
ERROR: unsatisfiable constraints:
  so:libcrypto.so.1.1 (missing):
    required by:
                 gdal-2.4.0-r0[so:libcrypto.so.1.1]
                 gdal-2.4.0-r0[so:libcrypto.so.1.1]
				 ...
```

I tried to point to the repository containing the libcrypto like this :  

```
    WORKDIR /usr/src/apps

    RUN apk add 'gdal=2.4.0-r0' --no-cache \
    --repository http://dl-cdn.alpinelinux.org/alpine/edge/community \
    --repository http://dl-cdn.alpinelinux.org/alpine/edge/main

   ...
```

But I still have an error, here after the full log of the build : 

```
Step 1/7 : FROM alpine:3.7
 ---> bc8fb6e6e49d
Step 2/7 : RUN echo "http://dl-cdn.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories
 ---> Using cache
 ---> e237b940e420
Step 3/7 : WORKDIR /usr/src/apps
 ---> Using cache
 ---> 83a6b0dce9bb
Step 4/7 : RUN apk add 'gdal=2.4.0-r0' --no-cache     --repository http://dl-cdn.alpinelinux.org/alpine/edge/community     --repository http://dl-cdn.alpinelinux.org/alpine/edge/main
 ---> Running in 929507c9a6d0
fetch http://dl-cdn.alpinelinux.org/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.7/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.7/community/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/edge/testing/x86_64/APKINDEX.tar.gz
(1/18) Installing libcrypto1.1 (1.1.1a-r1)
ERROR: libcrypto1.1-1.1.1a-r1: trying to overwrite etc/ssl/openssl.cnf owned by libressl2.6-libcrypto-2.6.5-r0.
(2/18) Installing ca-certificates (20190108-r0)
(3/18) Installing nghttp2-libs (1.36.0-r0)
(4/18) Installing libssh2 (1.8.0-r4)
(5/18) Installing libssl1.1 (1.1.1a-r1)
(6/18) Installing libcurl (7.63.0-r0)
(7/18) Installing libgcc (8.2.0-r2)
(8/18) Installing giflib (5.1.4-r2)
(9/18) Installing libjpeg-turbo (1.5.3-r4)
(10/18) Installing libpng (1.6.35-r0)
(11/18) Installing db (5.3.28-r1)
(12/18) Installing libsasl (2.1.27-r0)
(13/18) Installing libldap (2.4.47-r2)
(14/18) Installing libpq (11.1-r0)
(15/18) Installing sqlite-libs (3.26.0-r3)
(16/18) Installing libstdc++ (8.2.0-r2)
(17/18) Installing tiff (4.0.10-r0)
(18/18) Installing gdal (2.4.0-r0)
Executing busybox-1.27.2-r11.trigger
Executing ca-certificates-20190108-r0.trigger
1 error; 38 MiB in 31 packages
The command '/bin/sh -c apk add 'gdal=2.4.0-r0' --no-cache     --repository http://dl-cdn.alpinelinux.org/alpine/edge/community     --repository http://dl-cdn.alpinelinux.org/alpine/edge/main' returned a non-zero code: 1
```

Any idea ? 
Thank you a lot.

**Comment by quiqua** [comment]
I have not touch this in a while, the best option at the moment would be to upgrade to OSRM v5.21 and switch to a Debian based images.
Osmium-tools is available as a package for Debian and thus not required anymore to build from source.

My best guess is that something changed with the Alpine packages. Note that `gdal` is only available in the testing branches...

**Comment by tspoke** [comment]
Thanks for your suggestion. I moved to https://github.com/stefda/docker-osmium-tool/blob/master/Dockerfile and it works fine !

**Comment by quiqua** [comment]
@tspoke FYI: I just merged the migration to Debian and OSRM v5.22.0 if you want to check that out :)
